examples/heart_rate_monitor/main.c

```diff
@@ -265,7 +265,7 @@ static uint32_t advertising_start()
     ble_gap_adv_params_t adv_params;
 
     adv_params.type        = BLE_GAP_ADV_TYPE_ADV_IND;
-    adv_params.p_peer_addr = NULL;                           // Undirected advertisement.
+    adv_params.p_peer_addr = NULL;                        // Undirected advertisement.
```

Comment:
Well at least it's not a problem in this example since we are setting `adv_params.p_peer_addr = NULL;`. There is a difference with `ble_gap_addr_t` between the sd_api versions: https://github.com/NordicSemiconductor/pc-ble-driver/blob/master/src/sd_api_v3/sdk/components/softdevice/s132/headers/ble_gap.h#L461.